multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -2392,6 +2394,11 @@ def save(self, path: str, standalone: Optional[bool] = True):
                     path=path,
                 )
 
+        # Clear the documents cache dictionary before saving.
+        for modality in [DOCUMENT_IMAGE, DOCUMENT_PDF]:
```

Comment:
Should be only `DOCUMENT`？

Reply:
issues wrt saving will be addressed in a separate pr.